### DISCUSSION TITLE
A new idea for better discord theme

### DISCUSSION THREAD

**Discussion by Scorpions95** [Eight-P/BetterDiscord-Stuff#29] [Ideas]
Is good idea to create Linkin park theme for better discord?

with Lp backgroynd and some their icon and etc?

**Comment by Eight-P** [comment]
You can edit the theme file and change the background image by replacing the links.
I doubt it will look particularly good, but feel free to try.
I won't make any of that.

I gave another answer about changing the background [here](https://github.com/Eight-P/BD.8P/discussions/28#discussioncomment-10635643)
